addons/sale_timesheet/models/project_sale_line_employee_map.py

```diff
@@ -48,6 +52,16 @@ def _compute_currency_id(self):
         for line in self:
             line.currency_id = line.sale_line_id.currency_id if line.sale_line_id else False
 
+    @api.depends('employee_id.timesheet_cost')
+    def _compute_cost(self):
+        for line in self:
+            if not line.is_cost_changed:
+                line.cost = line.employee_id.timesheet_cost or 0.0
+
+    @api.onchange('cost')
+    def _onchange_cost(self):
+        self.is_cost_changed = self.employee_id and self.cost != self.employee_id.timesheet_cost
```

Comment:
If the user changes the `timesheet_cost` of his employee in the employee model then the `is_cost_changed` should not be recomputed in the case where the cost manually edited is equal to the newest timesheet cost of this employee ? 🤔 
Because it is not possible for the user to disable the manual edition.

Another solution would be to add the `is_cost_changed` (with`optional="hide"`) in the tree view of this model defined in the project form view.